### PR TITLE
KOGITO-101: Add Kube labels for rule endpoints

### PR DESCRIPTION
**Pull request by evacchi**
very simple change. Add rule unit labels to the collection, should be written in the output file, like processes.

### CODE REVIEW

File: kogito-codegen/src/main/java/org/kie/kogito/codegen/rules/RuleCodegen.java [hunk 1]
```diff
@@ -38,6 +38,10 @@
 import org.kie.kogito.codegen.di.DependencyInjectionAnnotator;
 import org.kie.kogito.codegen.rules.config.RuleConfigGenerator;
 
+/**
+ * @deprecated use {@link IncrementalRuleCodegen}
+ */
+@Deprecated
 public class RuleCodegen extends AbstractGenerator {
```
Comment (mswiderski) [inline kogito-codegen/src/main/java/org/kie/kogito/codegen/rules/RuleCodegen.java]: why not removing this class directly? Is there any use of it....

Comment (mswiderski) [inline kogito-codegen/src/main/java/org/kie/kogito/codegen/rules/RuleCodegen.java]: why not removing this class directly? Is there any use of it....

Comment (evacchi) [reply kogito-codegen/src/main/java/org/kie/kogito/codegen/rules/RuleCodegen.java]: good point. I left it around just for the Quarkus extension, but we might delete and rename. @mariofusco any objections?

Comment (evacchi) [reply kogito-codegen/src/main/java/org/kie/kogito/codegen/rules/RuleCodegen.java]: opening JIRA to remove class

**Pull request merged**